src/test/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpointUnitTest.java

```diff
@@ -53,7 +53,7 @@
   private static final String CASEID_UPRN_INCONSISTENT =
       "The UPRN of the referenced Case and the provided Address UPRN must be matching";
   private static final String CASEID_INCONSISTENT =
-      "The caseid in the modifyAddress URL does not match the caseid in the request body";
+      "The caseid in the given dto URL does not match the caseid in the request body";
```

Comment:
See above

Reply:
done

Reply:
"The path parameter caseId does not match the caseId in the request body" is clearer